requirements.txt

```diff
@@ -1,5 +1,25 @@
-Flask>=2.0.2
-Flask_Login>=0.5.0
-Flask_Migrate>=3.1.0
-Flask_SQLAlchemy>=2.5.1
-Werkzeug>=2.0.2
+alembic==1.7.5
+click==8.0.3
+dominate==2.6.0
+Flask==2.0.2
+Flask-Bootstrap==3.3.7.1
+Flask-JWT==0.3.2
+Flask-Login==0.5.0
+Flask-Migrate==3.1.0
+Flask-SQLAlchemy==2.5.1
+Flask-WTF==1.0.0
+greenlet==1.1.2
+importlib-metadata==4.10.0
+importlib-resources==5.4.0
+itsdangerous==2.0.1
+Jinja2==3.0.3
+Mako==1.1.6
+MarkupSafe==2.0.1
+PyJWT==1.4.2
```

Comment:
Are you sure you want all these requirements?

Reply:
This is all. Some are dependecies from others.

Reply:
I have reinstall the virtual enviroment and fix the requirerments.
The PyJWT was from the experimentation...
Some of the other packages are dependencies from Flask.
Is safe to remove them?